release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -411,6 +411,11 @@ rollout strategy to successfully upgrade an {product-title} cluster.
 * Experimental mode is now available, in which you may use Technology Preview features, such as creating applications using devfiles.
 * Technology Preview feature - support of devfiles is now available. To learn more, see the xref:../cli_reference/openshift_developer_cli/odo-release-notes.adoc#odo-notable-improvements_odo-release-notes[odo Release Notes].
 
+[id="ocp-4-4-openshift-pipelines-tp"]
+==== OpenShift Pipelines (Technology Preview)
+OpenShift Pipelines use Tekton custom resources to create extensible CI/CD solutions for automating deployments. These resources serve as building blocks to assemble the pipeline. OpenShift Pipelines provide a catalog of reusable Tasks that can be used to easily build pipelines. Each pipeline runs in an isolated container without requiring the maintenance of a CI server and is portable across multiple platforms.
```

Comment:
Custom Resources (CR)
These CRs .....the building blocks .... the Pipeline
build Pipelines
Each Pipeline